libs/block-mesh-manager/migrations/20240822174812_create_analytics_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE analytics
```

Comment:
missing indexes and the trigger i use in all tables to track changes
(might be not needed here)